docs/tutorials/porch-development-environment/bin/setup.sh

```diff
@@ -14,93 +14,107 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-os_type=$(uname)
-if [ "$os_type" = "Darwin" ]
-then
-  SED="gsed"
-else
-  SED="sed"
+porch_cluster_name=porch-test
+git_repo_name="$porch_cluster_name"
+gitea_ip=172.18.255.200  # should be from the address range specified here: https://github.com/nephio-project/porch/blob/main/docs/tutorials/starting-with-porch/metallb-conf.yaml
+self_dir="$(dirname "$(readlink -f "$0")")"
+
+function h1() {
+  echo
+  echo "** $*"
+  echo 
+}
+
+# check if git user.name is set in your environment
+if ! git config --global user.name > /dev/null ; then
+  cat << ENDOFEMSG
+Please set the user.name and user.email global configuration values in your git environment.
+This is needed for porch tests to run correctly. 
+E.g:
+  git config --global user.name "My Name"
+  git config --global user.email "myemail@example.com"
+ENDOFEMSG
+  exit 128
 fi
 
-# Create mgmt and edge1 clusters in kind
-curl -s https://raw.githubusercontent.com/nephio-project/porch/main/docs/tutorials/starting-with-porch/kind_management_cluster.yaml | \
-  kind create cluster --config=-
-
-curl -s https://raw.githubusercontent.com/nephio-project/porch/main/docs/tutorials/starting-with-porch/kind_edge1_cluster.yaml | \
```

Comment:
Not a major issue but I think the orig guide, which this is an extension of is using the egde1 cluster.
https://github.com/nephio-project/porch/tree/main/docs/tutorials/starting-with-porch#deploying-a-blueprint-onto-a-workload-cluster

Reply:
Yes, I was a bit uncertain regarding removing edge1. My rationale was that the "Starting with Porch" tutorial is more like a "User's Guide", while this script belongs to the "Setup the Developer Environment" guide. And I think edge1 is not needed to develop and test porch, it belongs more to ConfigSync. 
All-in-all I can add back edge1 if we think there is need for it during development.

Reply:
Ye I guess either way the README guides will need to be updated a bit.
May be best to get @liamfallon direction on it when he is back on 13/05